Rakefile

```diff
@@ -6,7 +6,7 @@ require "rake/testtask"
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList["test/**/test_*.rb"]
+  t.test_files = FileList["test/**/test_*.rb", "test/integration_test.rb"]
```

Comment:
It turned out we never actually ran the integration tests as they didn't match the file pattern, added them explicitly to the tests.

Reply:
Hah. IMO we should just rename the file to `test/test_integration.rb` so it matches the rest.

Reply:
works for me

Reply:
Done in cbdb45e31bf78e9aad930853feb033feb6e4c2e5